### ISSUE TITLE
Users can't edit submissions without 'edit all' permission

### ISSUE THREAD

**Issue by d-karstens** [verbb/formie#396]
**Description**
Users are unable to edit submissions when they have manage submission permissions for certain forms, they can view but the save button does not appear. When the user has "Edit all submissions" permission the save button appears, but they can also see submissions from every other form

**Steps to reproduce**
1. Create a user with manage permissions for a forms submissions, but no edit all permissions
2. View submissions on that form

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: No
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.22
- Craft version: 3.6.9
- Multi-site: No

**Additional context**

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.22",
  "...": "..."
}
```

Then run `composer update`.

**Comment by d-karstens** [comment]
Much appreciated!

**Comment by engram-design** [comment]
Fixed in [1.3.23](https://github.com/verbb/formie/releases/tag/1.3.23)
